### PR TITLE
Document wireguard reverse tunnel

### PR DESCRIPTION
**Pull request by asdil12**
Ticket: https://progress.opensuse.org/issues/168880

### CODE REVIEW

File: docs/Installing.asciidoc [hunk 1]
```diff
@@ -1552,6 +1552,91 @@ SSH and salt, e.g. to be used with
 https://github.com/os-autoinst/salt-states-openqa/, can be found in
 https://github.com/os-autoinst/openQA/blob/master/contrib/ay-openqa-worker.xml
 
+== Special network conditions
+There might be certain situations where the openQA workers cannot reach the openQA webui directly.
+In this case a reverse connection via SSH or wireguared might be useful
+allowing the openQA webui to connect to a worker opening a backchannel.
+
+=== Wireguared
```
Comment (kalikiana) [inline docs/Installing.asciidoc:1560]:
```suggestion
=== Wireguard
```

File: docs/Installing.asciidoc [hunk 2]
```diff
@@ -1552,6 +1552,91 @@ SSH and salt, e.g. to be used with
 https://github.com/os-autoinst/salt-states-openqa/, can be found in
 https://github.com/os-autoinst/openQA/blob/master/contrib/ay-openqa-worker.xml
 
+== Special network conditions
+There might be certain situations where the openQA workers cannot reach the openQA webui directly.
+In this case a reverse connection via SSH or wireguared might be useful
```
Comment (kalikiana) [inline docs/Installing.asciidoc:1557]:
```suggestion
In this case a reverse connection via SSH or wireguard might be useful
```

File: docs/Installing.asciidoc [hunk 3]
```diff
@@ -1552,6 +1552,91 @@ SSH and salt, e.g. to be used with
...
+[source,ini]
+--------------------------------------------------------------------------------
+[[fd0a::1]]
+key = FOO
+secret = BAR
+--------------------------------------------------------------------------------
+
+NOTE: The IPv6 address is written in square brackets as it is internally
+converted to a URL which requires this notation.
+This is also the reason why host specific the section headers need to
```
Comment (kalikiana) [inline docs/Installing.asciidoc:1638]:
```suggestion
This is also the reason why hosts specific to the section headers need to
```

File: docs/Installing.asciidoc [hunk 4]
```diff
@@ -1552,6 +1552,91 @@ SSH and salt, e.g. to be used with
 https://github.com/os-autoinst/salt-states-openqa/, can be found in
 https://github.com/os-autoinst/openQA/blob/master/contrib/ay-openqa-worker.xml
 
+== Special network conditions
+There might be certain situations where the openQA workers cannot reach the openQA webui directly.
+In this case a reverse connection via SSH or wireguard might be useful
+allowing the openQA webui to connect to a worker opening a backchannel.
```
Comment (Martchus) [inline docs/Installing.asciidoc:1558]: The wrapping of this paragraph is very inconsistent.

File: docs/Installing.asciidoc [hunk 5]
```diff
@@ -1552,6 +1552,91 @@ SSH and salt, e.g. to be used with
...
+
+=== Wireguard
+For wireguard using wg-quick is recommended.
```
Comment (Martchus) [inline docs/Installing.asciidoc:1561]:
```suggestion
For WireGuard using wg-quick is recommended.
```

File: docs/Installing.asciidoc [hunk 6]
```diff
@@ -1552,6 +1552,91 @@ SSH and salt, e.g. to be used with
 https://github.com/os-autoinst/salt-states-openqa/, can be found in
 https://github.com/os-autoinst/openQA/blob/master/contrib/ay-openqa-worker.xml
 
+== Special network conditions
+There might be certain situations where the openQA workers cannot reach the openQA webui directly.
+In this case a reverse connection via SSH or wireguard might be useful
+allowing the openQA webui to connect to a worker opening a backchannel.
+
+=== Wireguard
```
Comment (Martchus) [inline docs/Installing.asciidoc:1560]:
```suggestion
=== WireGuard
```

File: docs/Installing.asciidoc [hunk 7]
```diff
@@ -1552,6 +1552,91 @@ SSH and salt, e.g. to be used with
...
+[source,ini]
+--------------------------------------------------------------------------------
+[global]
+HOST=[fd0a::1]
+
+[[fd0a::1]]
+TESTPOOLSERVER = rsync://[fd0a::1]/tests
+--------------------------------------------------------------------------------
+
+Same for the `/etc/openqa/client.conf`
```
Comment (Martchus) [inline docs/Installing.asciidoc:1627]:
```suggestion
Same for `/etc/openqa/client.conf`:
```

**Review by kalikiana — APPROVED** [review]

**Review by Martchus — APPROVED** [review]

**Pull request merged**